packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -293,6 +295,7 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
         role={role}
         aria-label={ariaLabel}
         aria-disabled={disabled || undefined}
+        aria-description={ariaDescription}
```

Comment:
`aria-description` is not an Aria field. The only valid field is `aria-describedby`. If something supplies a 'description' we need to allocate an element with a unique id to contain it, and then set `aria-describedby` to be that id.